media/css/m24/grid.scss

```diff
@@ -2,61 +2,75 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-.m24-c-grid {
-    padding: $spacer-xl 0;
-}
 
-.m24-c-grid-cell {
-    padding-top: $spacer-lg;
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+// variables
+@import 'vars/color';
+@import 'vars/spacing';
+@import 'vars/text';
+@import 'vars/animation';
```

Comment:
**Suggestion:** It looks like you can remove the color and animation variables from here. Also, the grid variables moved to a new file you should include.

They're all included in the base file at the moment though so you technically don't need any here which is why this all works fine even though the grid variables moved.